perfkitbenchmarker/virtual_machine.py

```diff
@@ -285,36 +314,65 @@ def RemoteCopy(self, file_path, remote_path='', copy_to=True):
                     (retcode, full_cmd, stdout, stderr))
       raise errors.VmUtil.SshConnectionError(error_text)
 
+  def _PushLongRunningCommandScripts(self):
+    """Pushes the scripts required by LongRunningRemoteCommand to this VM.
+
+    If the scripts have already been placed on the VM, this is a noop.
+    """
+    with self._has_remote_command_script_lock:
+      if not self._has_remote_command_script:
+        for f in (EXECUTE_COMMAND, WAIT_FOR_COMMAND):
+          self.PushDataFile(f, os.path.join(vm_util.VM_TMP_DIR,
+                                            os.path.basename(f)))
+
   def LongRunningRemoteCommand(self, command):
     """Runs a long running command on the VM in a robust way.
 
+    Executes a long running command via a pair of scripts on the VM:
```

Comment:
[I know it was not clear in the old code before but I think this would be helpful to have:]

Consider adding a clarifying comment to indicate at which threshold the caller should consider a command as a "long running" command and invoke this function. e.g., 10 minutes, 30 minutes, or 1 hour?

Also, at a design level, AFAIK, this function should just work for all commands independent of execution duration (but it does come with a small overhead of async waiting, etc), I'm wondering why it can't be the default "run remote command function". Not blocking this CL but something to consider...